omod/src/main/webapp/template/localHeader.jsp

```diff
@@ -16,3 +16,4 @@
 <openmrs:htmlInclude file="/moduleResources/openhmis/cashier/js/init.js" />
 <openmrs:htmlInclude file="/moduleResources/openhmis/inventory/js/init.js" />
 <openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/curl.js" />
+
```

Comment:
actually the jquery has aready been loaded
if you view the page source you will c that we aready have a jquery loaded